keyboards/handwired/petruziamini/readme.md

```diff
@@ -0,0 +1,13 @@
+# handwired/petruziamini
+
+![handwired/petruziamini]
+
+My first handwired projec. My hoping is to learn how to use github and qmk firmware in one shot.
+
+* Keyboard Maintainer: [LLLKST](https://github.com/LLLKST)
+
+* Hardware Availability: *i will compile this later if it will work. I will also increase my level of english, i promise :)*
```

Comment:
> i will compile this later if it will work

You should be compiling locally to ensure your submission is valid and functions as expected.

Reply:
My mistake due to the translation from Italian. I meant that I would fill the readme file with information later. I compiled the code locally and it works. I have now made the other required changes as well. Thank you